.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.7"
```

Comment:
What about 2.6?

Reply:
yep, added. This is the first travis config I've written, so i was testing various setups out in the branch.